yourls/Makefile

```diff
@@ -0,0 +1,99 @@
+ROOT_DIR = ..
+include ${ROOT_DIR}/_scripts/Makefile.projects
+include ${ROOT_DIR}/_scripts/Makefile.instance
+
+.PHONY: config-hook
+config-hook:
+	@${BIN}/reconfigure_ask ${ENV_FILE} YOURLS_TRAEFIK_HOST "Enter the YOURLS domain name" yourls${INSTANCE_URL_SUFFIX}.${ROOT_DOMAIN}
+	@${BIN}/reconfigure ${ENV_FILE} YOURLS_INSTANCE=$${instance:-default}
+	@${BIN}/reconfigure_auth ${ENV_FILE} YOURLS
+	@echo
+	@${BIN}/reconfigure_ask ${ENV_FILE} YOURLS_USER "Enter the admin username for your YOURLS instance"
```

Comment:
I leave it up to you, but would you rather type a username and password here, or just have use admin and a random password?

Reply:
actually, does this use the YOURLS_PASS forever, or does it get copied into the config and then it no longer works if you change it? We dont want env files that cant be changed.

Reply:
There doesn't appear to be anything special about the "admin" user in YOURLS. You can have multiple users access the "admin" interface. The admin interface lets you have more control over the shortened URLs, as well as control over how YOURLS works in general. Since you might configure multiple users to access the "admin"interface, I think it's fine to let the "admin" username and password be user-defined instead of assigned by d.ry.